src/FuncSharp/Try/Try.cs

```diff
@@ -649,38 +585,21 @@ public A Get()
         {
             return Match(
                 s => s,
-                e =>
-                {
-                    var exception = e.SingleOption();
-                    if (exception.NonEmpty)
-                    {
-                        ExceptionDispatchInfo.Capture(exception.Get()).Throw();
-                    }
-
-                    throw new AggregateException(e);
-                }
+                e => throw e
```

Comment:
This will forget the stacktrace, previously the stack trace worked for single exception